pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -55,6 +55,14 @@ const (
 	Skip
 )
 
+const (
+	// NodeScoreMax is the max score which returned by the score plugin.
+	NodeScoreMax int = 100
```

Comment:
Any particular reason you preferred `NodeScoreMax` over `MaxNodeScore`?

Reply:
The KEP use the name `NodeScoreMax` and I also think golang has a preference to name const with "what it is" ahead, ex:

```go
const (
	VolumeStatusPending = "pending"
	VolumeStatusDone    = "done"
)
```

> However, the maximum count of blinking lights may be small compared to **NodeScoreMax**. To fix this, BlinkingLightScorer should also register for this extension point.

Reply:
I suggest we use MaxNodeScore, golang names limit values MaxInt, MaxFloat etc.

Reply:
> I suggest we use MaxNodeScore, golang names limit values MaxInt, MaxFloat etc.

sounds good